src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButton.tsx

```diff
@@ -54,23 +56,27 @@ const ArtworkActionsSaveButton: React.FC<ArtworkActionsSaveButtonProps> = ({
     isLiveOpen ||
     registrationAttempted
 
-  const openAuctionRegistrationPopover = useCallback(() => {
-    if (!ignoreAuctionRegistrationPopover) {
-      setPopoverVisible(true)
-    }
+  const maybeOpenAuctionRegistrationPopover = useCallback(() => {
+    if (ignoreAuctionRegistrationPopover) return
+
+    setPopoverVisible(true)
   }, [ignoreAuctionRegistrationPopover])
 
-  const onPopoverClosed = useCallback(() => {
+  const handleClose = useCallback(() => {
     setPopoverVisible(false)
   }, [])
 
+  const { value, clearValue } = useAuthIntent()
+
   useEffect(() => {
-    mediator.on("artwork:save", openAuctionRegistrationPopover)
+    // If we are coming in after authentication triggered by a save
+    // *maybe* open the auction registration popover
+    if (!value || value.action !== "save") return
```

Comment:
So you can see here: we wait for the value to change to something we're looking for...